integration/jnifuse/jnifuse-java/src/main/java/alluxio/jnifuse/utils/Environment.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.jnifuse.utils;
+
+public class Environment {
```

Comment:
Alluxio code convention:
```
public final class Environment {
  private Environment() {} // Prevent instantiation
```

Reply:
also, if this file is mostly derived from rocksdb lib, please acknowledge the original file in class comments

Reply:
I add a comment like `This file mostly derived from rocksdb lib.`, so we don't need to rename it to `EnvironmentUtils `?

AFAIK, there are another file named `alluxio.util.EnvironmentUtils` already.